es/index.js

```diff
@@ -16,4 +16,10 @@ if (typeof self !== 'undefined') {
 }
```

Comment:
Probably not for this PR, but I suggest adding a `globalThis` branch to the above if-else sequence.